# 
# @brief    : custom mavlink msg, message id="155" name="YAW_SP_CALCULATED_M2P"
# @time     : 2018/09/13
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
# 

#Information about component grabbing

std_msgs/Header header

uint64 timestamp   #timestamp
unit8  grab_status #whether grab the box